x-pack/platform/packages/shared/kbn-streamlang/types/processors/ingest_pipeline_processors.ts

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
This is just copied over verbatim from the `streams-schema` package for now, the schema package will be cleaned up once we've completed the conversion process to Streamlang (for the APIs, UI etc).